app/(playground)/p/[agentId]/beta-proto/graph/provider.tsx

```diff
@@ -52,7 +52,9 @@ export const GraphProvider: FC<PropsWithChildren<GraphProviderProps>> = ({
 	useEffect(() => {
 		if (isInitialMount.current) {
 			isInitialMount.current = false;
+			console.log({ a: state.graph });
 		} else {
+			console.log({ b: state.graph });
```

Comment:
nits: is this necessary?

Reply:
Sorry, I made a commit by mistake.
Fixed in commit ea2d0311e246e323e2a1bde7ab12ef33ba1ffab0